SECURITY.md

```diff
@@ -0,0 +1,31 @@
+# Security
+
+If you believe you have found a security vulnerability in Slicer, please report it to us as described below.
```

Comment:
Probably a good idea. Are security vulnerability in Slicer extensions included in the scope? Or should users report security issues of extensions through different means?

Reply:
Good question.  I think the correct answer is that we have limited influence on the extension developers and claim no responsibility for their actions.  We might, as a best effort, pass on any reports but we can't claim to be doing anything rigorous (in fact, it might be that we don't do anything, e.g. if nobody notices the report or someone thinks someone else will handle it).

Reply:
We could only attempt to make commitments  for selected extensions - that we develop ourselves, or we know the developers of, or at least the developers are known to be responsive. For this, we would need to do what we have been planning for years: specify "support level" or "stability" for each extension (at least 2 but maybe 3 levels).

Reply:
I don't think we should be making any "commitments" in a sense that people would interpret as legally binding, but we can say we intend to make an effort.

Reply:
A more direct example would be Slicer extensions under the "Slicer" GitHub organization. 

These 2 Slicer extensions are under the "Slicer" GitHub organization and are built-in to Slicer currently. Security reports related to this code falls under the scope of this SECURITY.md document where reports should go to the discourse email?
https://github.com/Slicer/LandmarkRegistration
https://github.com/Slicer/SlicerSurfaceToolbox

These are additional Slicer extensions under the "Slicer" GitHub organization, but are not built-in to Slicer and are installed as extensions through the extensions manger. Do these also fall under the scope of this SECURITY.md?
https://github.com/Slicer/SlicerNeuro
https://github.com/Slicer/SlicerJupyter

Reply:
I would not expected SECURITY.md document in Slicer/Slicer would propagate to Slicer/other repositories, but it does not hurt to clarify this. Later we can add SECURITY.md documents to other repositories.

Reply:
You wouldn't expect SECURITY.md to apply to https://github.com/Slicer/LandmarkRegistration and https://github.com/Slicer/SlicerSurfaceToolbox even though it is included in the binaries at https://download.slicer.org/?

Reply:
> I would not expected SECURITY.md document in Slicer/Slicer would propagate to Slicer/other repositories

This could be addressed by creating a repository called `.github`  where we would include the file `SECURITY.md`

See https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

Alternatively, since the Slicer project is our main code base, the default `SECURITY.md` could simply include a link to the one of Slicer.

Reply:
That said, this could easily be done in follow-up step.